packages/jumpstarter/jumpstarter/client/lease.py

```diff
@@ -39,6 +40,7 @@ class Lease(AbstractContextManager, AbstractAsyncContextManager):
     release: bool = True  # release on contexts exit
     controller: jumpstarter_pb2_grpc.ControllerServiceStub = field(init=False)
     tls_config: TLSConfigV1Alpha1 = field(default_factory=TLSConfigV1Alpha1)
+    grpc_options: dict[str, Any] = field(default_factory=dict)
```

Comment:
```suggestion
    grpc_options: dict[str, str | int] = field(default_factory=dict)
```

It seems all the options are either str or int.